Multiple components on a page breaks the dev server

**Describe the bug**
Using 2 or more component shortcodes on the same page with hydration (ex. 2 eagerly loaded components) breaks on server startup. Note: this does _not_ happen when you start with a single shortcode on the page, then add another shortcode **while the dev server is running.** It will only happen when:
- multiple shortcodes are present on the page before running `slinkity --serve`
- The Vite cache is fresh (i.e. `node_modules/.vite` is empty)

**Error**
```sh
`TemplateContentRenderError` was thrown

Eleventy:EleventyErrorHandler (error stack):
TemplateContentRenderError: Having trouble rendering liquid template ./src/index.md

[11ty] tag "react" not found, file:./src/index.md, line:3, col:1

`ParseError` was thrown

Eleventy:EleventyErrorHandler (error stack):
tag "react" not found, file:./src/index.md, line:3, col:1

   1| # ajcwebdev-slinkity
   2| 
>> 3| {% react 'components/Hello' %}
   4| 
   5| {% react 'components/Counter' %}

Eleventy:EleventyErrorHandler ParseError:

tag "react" not found, file:./src/index.md,
line:3, col:1 at ParseError.LiquidError [as constructor]

(/Users/ajcwebdev/ajcwebdev-slinkity/node_modules/liquidjs/dist/liquid.node.cjs.js:577:28)

[11ty] tag "react" not found

`AssertionError` was thrown:

Eleventy:EleventyErrorHandler (error stack):
AssertionError: tag "react" not found at new AssertionError (/Users/ajcwebdev/ajcwebdev-slinkity/node_modules/liquidjs/dist/liquid.node.cjs.js:653:28)
```

**To Reproduce**
Steps to reproduce the behavior:
1. Create any page template with multiple shortcodes:

```njk
{% react 'Component' %}
{% react 'Component2' %}
```

2. Start the dev server using `slinkity --serve` or `slinkity --serve --incremental`

Notice an error occurs.


**Expected behavior**
The build should continue without issues.

**Additional context**
The error hints at using the same import name across each hydration loader. This shouldn't cause any issues since they're wrapped in different `script` tags, but it looks like ESBuild smashes the script tags together during parsing.

Investigated using the latest `v0.13` for ESBuild and downgrading to Vite `v2.3`, but the issue seems to persist.

--- a/src/plugin/reactPlugin/2-pageTransform/toLoaderScript.js
+++ b/src/plugin/reactPlugin/2-pageTransform/toLoaderScript.js
@@ -18,20 +18,20 @@ module.exports = function toLoaderScript({ componentPath, hydrate, id, props = {
   const componentImportStatement = JSON.stringify(toUnixPath(componentPath))
   if (hydrate === 'eager') {
     return `<script type="module">
-    import Component from ${componentImportStatement};
-    import eagerLoader from ${toClientImportStatement('_eager-loader.js')};
+    import Component${id} from ${componentImportStatement};
+    import eagerLoader${id} from ${toClientImportStatement('_eager-loader.js')};
   
-    eagerLoader({ 
+    eagerLoader${id}({ 
       id: "${id}",
-      Component,
+      Component: Component${id},
       props: ${stringify(props)},
     });
   </script>`
   } else if (hydrate === 'lazy') {
     return `<script type="module">
-    import lazyLoader from ${toClientImportStatement('_lazy-loader.js')};
+    import lazyLoader${id} from ${toClientImportStatement('_lazy-loader.js')};
   
-    lazyLoader({ 
+    lazyLoader${id}({ 
       id: "${id}",
       componentImporter: async () => await import(${componentImportStatement}),
       props: ${stringify(props)},

